### PR TITLE
Fix up .p-link—external

### LINKED ISSUE #720: External icon doesn't match design spec
How the external icon should look:

screen shot 2016-12-12 at 15 24 45

**Update @yaili:** See design spec https://github.com/ubuntudesign/vanilla-design/tree/master/Links

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

- Raise icon off the baseline
- Change color of icon on link hover

## QA

- Pull code
- Run `gulp jekyll`
- Check [example](http://127.0.0.1:4325/vanilla-framework/examples/patterns/links/links-external/) against [design spec](https://github.com/ubuntudesign/vanilla-design/blob/master/Links/links.png)

## Details

Fixes #720

### CODE REVIEW

**Comment by anthonydillon** [comment]
I think we need switch back from using after to a background element as an after element wraps.

For reference:
http://codepen.io/anthonydillon/pen/aJdzpV

File: package.json [hunk 1]
```diff
@@ -42,5 +42,5 @@
   "scripts": {
     "clean": "rm -r build; rm -r node_modules"
   },
-  "version": "1.1.6"
+  "version": "1.1.7"
```
Comment (anthonydillon) [inline package.json]: Was this missed in the last release?

**Comment by anthonydillon** [comment]
The standard external link looks good but seems to be missing the strong-external styling.

Not sure how you should do that? `p-link--strong p-link--external`?

**Comment by barrymcgee** [comment]
Related: https://github.com/vanilla-framework/vanilla-framework/issues/925

**Comment by anthonydillon** [comment]
Looks like there is a conflict in this branch.

File: scss/_patterns_links.scss [hunk 2]
```diff
@@ -6,9 +6,12 @@
     }
 
     // Used for links point at a different domain
-    &--external::after {
-      @include link-svg-icon;
-      margin-left: .25em;
+    &--external {
+      @include link-svg-icon($color-link);
```
Comment (anthonydillon) [inline scss/_patterns_links.scss]: This will need to be on an after element or a background image as the content is replacing the content of the link.

**Comment by anthonydillon** [comment]
The strong external link is turning blue of hover. The design spec states it should remain dark.

**Comment by anthonydillon** [comment]
The link color is still changing to blue on hover.

**Pull request merged**